From c2ad3abe37b101823fb47776c0219330cd1ce830 Mon Sep 17 00:00:00 2001
From: Rabeeh Khoury <rabeeh@solid-run.com>
Date: Tue, 8 Jul 2014 00:02:28 +0300
Subject: [PATCH 1/2] bcm80211 driver on kernel 3.0.35 is loaded once

There are issues with insmod / rmmod this driver on LK 3.0.35.
So keep it loaded on boot time all the way

Signed-off-by: Rabeeh Khoury <rabeeh@solid-run.com>
---
 wifi/wifi.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/wifi/wifi.c b/wifi/wifi.c
index c363c14..dffc443 100644
--- a/wifi/wifi.c
+++ b/wifi/wifi.c
@@ -119,7 +119,7 @@ static int insmod(const char *filename, const char *args)
     void *module;
     unsigned int size;
     int ret;
-
+    return 0; /* bcm80211 driver in LK 3.0.35 loads at at boot time */
     module = load_file(filename, &size);
     if (!module)
         return -1;
@@ -135,7 +135,7 @@ static int rmmod(const char *modname)
 {
     int ret = -1;
     int maxtry = 10;
-
+    /* bcm80211 driver in LK 3.0.35 will never unload */
     while (maxtry-- > 0) {
         ret = delete_module(modname, O_NONBLOCK | O_EXCL);
         if (ret < 0 && errno == EAGAIN)
-- 
1.7.9.5

